qiskit/providers/aer/backends/cluster/exceptions.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Exception for the Job Manager modules."""
+
+from qiskit import QiskitError
+
+
+class AerClusterError(QiskitError):
```

Comment:
Not sure we need so many additional error classes. Can't we just use `AerError` for all of them?